mission/functions/systems/vehicle_asset_manager/server/fn_veh_asset_marker_create.sqf

```diff
@@ -0,0 +1,59 @@
+/*
+	File: fn_veh_asset_marker_create.sqf
+	Author: Savage Game Design
+	Public: No
+
+	Description:
+		Creates a marker of the specific type for every player, localized. Replaces any previous markers.
+
+	Parameter(s):
+		_spawnPoint - Spawn point to create the marker for [HashMap]
+		_type - Type of marker [String]
+
+	Returns: nothing
+
+	Example(s): none
+*/
+
+
+params ["_spawnPoint", "_type"];
+
+private _vehicle = _spawnPoint getOrDefault ["currentVehicle", objNull];
+
+private _textGenerator = [
+	[_type, _vehicle],
+	{
+		params ["_type", "_vehicle"];
+		private _markerPrefix = "";
+		if (_type == "WRECK") then {
+			_markerPrefix = localize "STR_vn_mf_wreck";
+		};
+
+		if (_type == "IDLE") then {
+			_markerPrefix = localize "STR_vn_mf_idle";
+		};
+
+		if (_type == "DISABLED") then {
+			_markerPrefix = localize "STR_vn_mf_disabled";
+		};
+
+		format [
+			"%1: %2",
+			_markerPrefix,
+			getText (configFile >> "CfgVehicles" >> typeOf _vehicle >> "displayName")
+		]
+	}
+];
+
+private _markerName = format ["marker_%1", _spawnPoint get "id"];
+[_markerName, getPos _vehicle, _textGenerator] call para_g_fnc_create_localized_marker;
```

Comment:
I think there could be a conflict in marker names that needs to be resolved. i'm 97% sure when placing down a map marker in eden editor, Arma will name it using this same convention `marker_1` etc.

I think we may want to change this 
```
private _markerName = format ["marker_%1", _spawnPoint get "id"];
```

to something like
```
private _markerName = format ["vehicle_live_marker_%1", _spawnPoint get "id"];
```

Reply:
confirmed

```
C:\Users\dijksterhuis\Documents\MIKE-FORCE\Mike-Force\maps\cam_lao_nam\mission.sqm:
 13289  			dataType="Marker";
 13290  			position[]={17184.65,14.97998,7022.7148};
 13291: 			name="marker_100";
 13292  			text="SP1";
 13293  			type="mil_warning";
 .....
 13913  			dataType="Marker";
 13914  			position[]={20067.029,1.4619141,9779.8848};
 13915: 			name="marker_101";
 13916  			text="P2";
 13917  			type="mil_pickup";
 .....
 13925  			dataType="Marker";
 13926  			position[]={20083.482,1.1785889,9773.7529};
 13927: 			name="marker_103";
 13928  			text="P3";
 13929  			type="mil_pickup";
 .....
 15777  			dataType="Marker";
 15778  			position[]={17423.734,30.685818,5639.2544};
 15779: 			name="marker_124";
 15780  			text="Rearm/Repair";
 15781  			type="b_maint";
 .....
```

Reply:
for reference


### boats are idle -- shows strings from other map marks

![20240406151730_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/23362ff7-0d4c-4f07-840c-89d2da599104)

### player in boat -- shows the correct string

![20240406172013_1](https://github.com/Bro-Nation/Mike-Force/assets/11841332/6bffea7f-0a7a-4fd5-a8a7-9cffa247a8d8)

Reply:
✅ RESOLVED